packages/block-library/src/group/style.scss

```diff
@@ -2,3 +2,8 @@
 	// This block has customizable padding, border-box makes that more predictable.
 	box-sizing: border-box;
 }
+
+// We need this so groups with negative margins overlap as expected.
+:where(.wp-block-group.wp-block-group-is-layout-constrained) {
```

Comment:
Is the doubling-up of class names necessary? Also, should the modifier syntax not be `.wp-block-group--is-layout-constrained` according to BEM principles?

Reply:
Mmmh, the specificity is the same with one or two classes, if we use only the second, I am unsure if that class can be anywhere else but a group block but I prefer to keep it safe. And about the BEM principles, I'm not sure if we are following them or not but that is a class added by the layout code and changing it now is probably more disruptive at this point.